app/interactors/send_timer_reminders.rb

```diff
@@ -0,0 +1,18 @@
+class SendTimerReminders
+  include Interactor
+  include HasHarvest
+
+  before do
+    context.date ||= Date.current
+  end
+
+  def call
+    User.all.each do |user|
+      next if user.timer_reminder_sent_on.try(:today?)
```

Comment:
`User.where("timer_reminder_sent_on < ? OR timer_reminder_sent_on IS NULL", Date.current).joins(:days).where(days: {pto: false})`